### ISSUE TITLE
Wrong file bitboard initializers

### ISSUE THREAD

**Issue by SloPro** [rudzen/ChessLib#22]
These constants in Bitboards.cs are wrong:
```
private const ulong FILEC = 0x2020202020202020;
private const ulong FILED = 0x1010101010101010;
private const ulong FILEE = 0x808080808080808;
private const ulong FILEF = 0x404040404040404;
```
They should be
```
private const ulong FILEC = 0x404040404040404;
private const ulong FILED = 0x808080808080808;
private const ulong FILEE = 0x1010101010101010;
private const ulong FILEF = 0x2020202020202020;
```

**Comment by rudzen** [comment]
I'll look into it asap.

**Comment by rudzen** [comment]
Confirmed. Fixed in #20 

Pending merge.
